packages/model/src/user.ts

```diff
@@ -8,3 +8,8 @@ export interface Invitee {
   orcid?: string;
   institution?: string;
 }
+
+export interface UserResponse extends Invitee {
+  id: string;
+  connections?: [{ code: string }];
```

Comment:
```suggestion

```

We don't want to expose the connection until we have a better understanding on how we'll map multiple accounts.

Reply:
But that's not how TypeScript works, we need to declare it or the backend e.g. in `create-user.ts` will error because it's trying to set the property.

Reply:
In that case `create-user.ts` is exposing a property it shouldn't.